docs/running_js_scripts.md

```diff
@@ -60,3 +85,11 @@ For more information about this example, please see: [running async/await script
 ```
 
 For more script examples, please see [Frequently Asked Scripts](FAS.html).
+
+## `require` in scripts at Remix
+
+`require` statement is supported in limited manner for remix supported modules with Remix Scripts.
```

Comment:
the sentence above should be: (I added an a before limited)(and changed the case of Remix)
`require` statement is supported in a limited manner for Remix supported modules with Remix Scripts.

Reply:
updated